lib/cstruct.ml

```diff
@@ -27,10 +27,11 @@ type buffer = (char, Bigarray.int8_unsigned_elt, Bigarray.c_layout) Bigarray.Arr
  * indeed maintains this invariant.
  *)
 
+(* Note that offset is stored in both bits and bytes! *)
```

Comment:
Feel free to rename the field as off_bits or encapsulate this in an abstract size type - that's better than a comment in my mind

Reply:
At the end, this update adds lot of headache when `{get,set}_uint{8,16,32,64}` did not changes and believed on `off` concerns bytes (and not bits).

For example, `memset` divides by 8 `off`, I don't understand why.

And it seems that any `get_bits` requires an absolute `off` as an argument (and did not rely on internal `off` offset).

Reply:
I can change these functions to use `t.off` if you think it is better, that's not a problem.